login.css

```diff
@@ -110,7 +110,7 @@ p{
 	margin-left:130px;
 	margin-top:250px;
 	font-size:24px;
-	color:black;
+	color:#0C59CF;
```

Comment:
06/08/2021: I make the color for the heading to be navy blue because it was one of my main color in my music database and also will help to show the main focus that user need to look for.